nav2_bringup/bringup/launch/tb3_simulation_launch.py

```diff
@@ -93,7 +93,7 @@ def generate_launch_description():
         'default_bt_xml_filename',
         default_value=os.path.join(
             get_package_share_directory('nav2_bt_navigator'),
-            'behavior_trees', 'navigate_w_replanning_and_recovery.xml'),
+            'behavior_trees', 'follow_point.xml'),
```

Comment:
Don't change the default BT

Reply:
Yes, sorry. I didn't want to add this fail to the commit. I will undo in the next commit.